Client/src/components/App.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class App extends React.Component {
+    constructor(props) {
```

Comment:
I would suggest changing your VS code settings tabsize to 2 instead of 4. Some of your indentations are 4 spaces as opposed to 2.